src/pages/signin/SignInPage.js

```diff
@@ -76,7 +79,10 @@ class SignInPage extends Component {
         return (
             <>
                 <SafeAreaView style={[styles.signInPage]}>
-                    <SignInPageLayout>
+                    <SignInPageLayout
+                        showWelcomeText={showLoginForm}
+                        showWelcomeScreenshot={showLoginForm}
```

Comment:
I think these should be `shouldShowWelcomeText` and `shouldShowWelcomeScreenshot`.